enterprise/install-scripts/openshift.ks

```diff
@@ -3446,6 +3456,20 @@ declare -A valid_settings=( [CONF_ABORT_ON_UNRECOGNIZED_SETTINGS]= [CONF_ACTIONS
   # The list of packages to install.
   cartridges="${CONF_CARTRIDGES:-standard}"
 
+  jbosseap_version="${CONF_JBOSSEAP_VERSION:-6.3}"
+  # Check jbosseap_version for validity
+  case "${jbosseap_version}" in
+      (6.3|6.4)
+          jbosseap_version="${jbosseap_version}"
+          jbosseap_yumvalidator_role="node-eap-${jbosseap_version}"
+          ;;
+      (current|6)
+          jbosseap_version="6"
+          jbosseap_yumvalidator_role="node-eap"
+          ;;
+    (*) abort_install "Unrecognized JBoss EAP channel version: ${CONF_JBOSSEAP_VERSION}" ;;
```

Comment:
I mean, `CONF_JBOSSEAP_VERSION` is used both to derive the channel name and to derive the `oo-admin-yum-validator` role, so it's not just the *channel* version.  No big deal though.